src/Core/Asset/AnimationTime.hpp

```diff
@@ -0,0 +1,143 @@
+#ifndef RADIUMENGINE_ANIMATION_TIME_HPP
+#define RADIUMENGINE_ANIMATION_TIME_HPP
+
+#include <Core/RaCore.hpp>
+#include <algorithm>
+#include <limits>
+
+namespace Ra {
+namespace Core {
+namespace Asset {
+
+/**
+ * Defines the type for time values.
+ */
+using Time = Scalar;
+
+/**
+ * The AnimationTime class stores the start and end times of an animation.
+ */
+class RA_CORE_API AnimationTime
+{
+  public:
+    AnimationTime( const Time& start = std::numeric_limits<Time>::max(),
```

Comment:
Do not pollute the namespace..

```suggestion
/**
 * The AnimationTime class stores the start and end times of an animation.
 */
class RA_CORE_API AnimationTime
{
  public:
    using Time = Scalar;
    AnimationTime( const Time& start = std::numeric_limits<Time>::max(),
```